### PR TITLE
rest v1 spot & futures update

### PR DESCRIPTION
**Pull request by t0chk**
- add plan order symbol canceling
- fix candles data params (spot & futures)
- fix futures filter data (data in api-doc dither from real request!

### CODE REVIEW

File: src/spot-client.ts [hunk 1]
```diff
@@ -111,12 +112,16 @@ export class SpotClient extends BaseRestClient {
   getCandles(
     symbol: string,
     period: SpotKlineInterval,
-    pagination?: Pagination,
-  ): Promise<APIResponse<any>> {
+    limit?: string,
+    after?: string,
+    before?: string,
```
Comment (tiagosiebler) [inline src/spot-client.ts:117]: Hmm this is a breaking change, please revert it back to:
```typescript
    pagination?: Pagination,
```

File: src/types/request/v1/futuresV1.ts [hunk 2]
```diff
@@ -215,3 +215,5 @@ export interface HistoricPlanOrderTPSLRequest {
   isPre?: boolean;
   isPlan?: string;
 }
+
+export type FuturesCandleData = string[6];
```
Comment (tiagosiebler) [inline src/types/request/v1/futuresV1.ts:219]: Would love a small tsdoc-style comment to say where in the array each candle prop is

File: src/types/response/futures.ts [hunk 3]
```diff
@@ -32,21 +32,30 @@ export interface FuturesAccount {
...
+  sellLimitPriceRatio: string
+  sizeMultiplier: string
+  supportMarginCoins: string[]
+  symbol: string
+  symbolName: string
+  symbolStatus: string
+  symbolType: string
+  takerFeeRate: string
+  volumePlace: string
 }
```
Comment (tiagosiebler) [inline src/types/response/futures.ts:59]: Nitpick (sorry), semicolons missing. You can run the linter and it'll add them back automatically (as well as other misc formatting things)

**Review by tiagosiebler — DISMISSED** [review]
Looks good, some smaller changes

**Comment by t0chk** [comment]
its all done!

**Review by tiagosiebler — APPROVED** [review]
Looks great, thanks!

**Pull request merged**